python/ray/train/batch_predictor.py

```diff
@@ -204,6 +205,7 @@ def calculate_accuracy(df):
         class ScoringWrapper:
             def __init__(self):
                 checkpoint = ray.get(checkpoint_ref)
+                predictor_kwargs = ray.get(predictor_kwargs_ref)
```

Comment:
don't we need a ray.put somewhere?

Reply:
It's on line 39.

I tried to follow the pattern from `self.checkpoint_ref` and call `ray.put` in the constructor.